Appendix/Initial locking section (#23)
MrLongbottom/P10-Article: sections/appendix/locking.tex

\subsection{Locking the gibbs sampler}\label{sec:appendix/locking}

> Normally when working with topic modelling, one does know which topics will be present in the document set before training the model.

Reviewer: ```suggestion
Normally when working with topic modeling, one does not know which topics will be present in the document set before training the model.
```

However, the taxonomy metadata fields provides some general subject names of different levels of abstraction and some amount of documents attached to these subject names.

> This provides a unique opportunity of using the existing taxonomies as higher-level topics.

Reviewer: ```suggestion
This provides a unique opportunity for using the existing taxonomies as higher-level topics.
```

The pachinko model normally samples, like the Gibbs sampler, each taxonomy level and estimates the different topics for each document.

However, in our case, we have a partially observed variable and we want to use the existing meta-information to estimate the topics quicker and more accurately.

> When we sample in the \gls{pam}, we only sample the unobserved variables and let the observed variables be. 

Author: ```suggestion
When we sample in the \gls{pam}, we only sample the unobserved taxonomies and let the observed taxonomies be. 
```

> This has the effect of letting the unobserved variable fit the existing topics within the model and making the process of sampling faster.

Author: ```suggestion
This has the effect of letting the unobserved taxonomy fit the existing topics within the model and making the process of sampling faster.
```

> However, locking these variables into places also hinders further improvement to the observed variables, since they do not change during training.   

Author: ```suggestion
However, locking these taxonomies into places also hinders further improvement to the observed taxonomies, since they do not change during training.   
```

\subsection{Locking the gibbs sampler}\label{sec:appendix/locking}

> The pachinko model normally samples, like the gibbs sampler, each taxonomy level and estimates the different topics for each document.

Author: ```suggestion
The pachinko model normally samples, like the Gibbs sampler, each taxonomy level and estimates the different topics for each document.

```

\subsection{Locking the gibbs sampler}\label{sec:appendix/locking}

> The pachinko model normally samples, like the Gibbs sampler, each taxonomy level and estimates the different topics for each document.

Reviewer 2: ```suggestion
Normally when working with topic modelling, one does know which topics will be present in the document set before training the model.
However, the taxonomy metadata fields provides some general subject names of different levels of abstraction and some amount of documents attached to these subject names.
This provides a unique opportunity of using the existing taxonomies as higher-level topics.

The pachinko model normally samples, like the Gibbs sampler, each taxonomy level and estimates the different topics for each document.
```

prøvede at adde en intro som gav lidt mere context

> However, in our case, we have a partially observed variable and we want to use the existing meta-information to estimate the topics quicker and more accurately.

Reviewer: Generelt i teksten når 'variable' bliver brugt, er jeg usikker på hvad 'variable' er.

Reviewer 2: der menes taxonomy. Tror også det er bedre bare at skrive taxonomy.

Author: ```suggestion
However, in our case, we have a partially observed taxonomy and we want to use the existing meta-information to estimate the topics quicker and more accurately.
```

> When we sample the pachinko, we only sample the unobserved variables and let the observed variable be. 

Reviewer: ```suggestion
When we sample in the \gls{pam}, we only sample the unobserved variables and let the observed variables be. 
```

This has the effect of letting the unobserved variable fit the existing topics within the model and making the process of sampling faster.

> However, when locking these variables into places also hinders further improvement to the observed variables, since they do not change during training.   

Reviewer: ```suggestion
However, locking these variables into places also hinders further improvement to the observed variables, since they do not change during training.   
```
